HACKING.md

```diff
@@ -277,6 +117,74 @@ Or in order to log starting at page reload:
 
     > window.sessionStorage["debugging"] = "channel"
 
+## Building Cockpit binaries
+
+For more complex hacking on Cockpit beyond the user interface, you need to
+build the Cockpit binaries locally and install the relevant dependencies.
+Currently, recent x86_64 architectures of Fedora are most often used for
+development.
+
+Check `tools/cockpit.spec` for the concrete Fedora build dependencies.
+The following should work in a fresh Git clone:
+
+    $ sudo yum-builddep tools/cockpit.spec
```

Comment:
I think we should use dnf here, since that is recommended and yum deprecated:
```
    $ sudo dnf builddep tools/cockpit.spec
    $ sudo dnf install nodejs npm
```